### DISCUSSION TITLE
How to add two Receivers with different antenna arrays to a scenario?

### DISCUSSION THREAD

**Discussion by Junjie-he-01** [NVlabs/sionna#658] [General]
I want simulate a ISAC system so i want to add two Receivers with different antenna arrays as sensing rx and communication rx. As shown in the figure, it doesn't work; your example jupyter seems to only set the same antenna array for all Receivers. Need your help,thanks. And something wrong in the figure
![Snipaste_2024-11-12_18-57-28](https://github.com/user-attachments/assets/ee5ea426-54f8-4839-bf3c-93ecfc289962)
![Snipaste_2024-11-12_18-50-32](https://github.com/user-attachments/assets/ffdf4e82-add3-4efd-9b5b-9cd1d21f7d24)

**Comment by faycalaa** [comment]
Hi @JJwifisensing ,

Sionna supports only the use of a single antenna array shared by all receivers.

**Reply by potatoess11** [reply]
sionna supports to change or set antenna gain?

**Comment by Junjie-he-01** [comment]
> Hi @JJwifisensing ,
> 
> Sionna supports only the use of a single antenna array shared by all receivers.

Means that we can only set antenna array for all receiver. If i want to build a ISAC system, the sensing receiver must be same with the communication receiver, single antenna array or multi antenna array. Oh that's so bad, MIMO is difficult for me.

**Comment by faycalaa** [comment]
If you only need to use a single antenna for communication, maybe that you could set up an antenna array for the receiver and then select just one antenna from the channel impulse response to run link-level simulations of communications.

Hope this helps.

**Comment by Junjie-he-01** [comment]
> If you only need to use a single antenna for communication, maybe that you could set up an antenna array for the receiver and then select just one antenna from the channel impulse response to run link-level simulations of communications.
> 
> Hope this helps.

Thanks a lot! I will try it，it‘s a nice suggestion. I can set up an array for two receivers but select one antenna for communications. Hope Sionna can be better. thanks
